module-2/backend/week_two.md

```diff
@@ -26,20 +39,82 @@ end
 
 Now how would you find the owner of the team with an id of 4?
 
+@team.owners.where(owner_id: 4)
```

Comment:
Go back and review this! Want to have this down cold.

Adding the code above allows us to do the following:

```ruby
Team.find(4).owner
```

Reply:
Same mistake with the same assumption I made in question 3.